src/ConnectedMode/UI/ProjectSelection/ProjectSelectionViewModel.cs

```diff
@@ -19,15 +19,22 @@
  */
 
 using System.Collections.ObjectModel;
+using SonarLint.VisualStudio.ConnectedMode.UI.Resources;
 using SonarLint.VisualStudio.Core.WPF;
 
 namespace SonarLint.VisualStudio.ConnectedMode.UI.ProjectSelection;
 
-public class ProjectSelectionViewModel : ViewModelBase
+public class ProjectSelectionViewModel(
```

Comment:
I remember that this topic was brought up in our meeting about coding conventions: that we should prefer explicit constructors.

However, I can't find this point on the repo docs 🤔

I would suggest to revert that change, until it's clear that we want to go that way.

Reply:
I don't remember this discussion. I would rather skip this comment as I think using implicit constructors is not harmful, and we are already doing it in new classes.

Reply:
Implicit constructors are not harmful, after all, it's a C# feature. It's also true that there are other places in our codebase that are used (definitely they are not the majority though) and there might be good reasons to use them.

My point here, is that the whole team should be aligned on using them or not to keep the code consistent. It is a gray area as it is not documented anywhere yet. My suggestion remains, that it's better to revert the change until we are all aligned. 🤷

Reply:
Sorry, @vnaskos-sonar. I did not revert the change as I was not sure if it was worth the effort. If I revert it now and then, in the future, we decide implicit constructors are fine, then it would be yet additional effort to revert it back.

Also, Resharper recommends using the implicit constructor, so having the squiggly line and ignoring it is kind of hard.